dependencies/FindOpenMP.cmake

```diff
@@ -1,26 +0,0 @@
-find_library(OpenMP_LIBRARY
```

Comment:
I included this just to make it work on the Mac. It will probably no longer work since detection of OpenMP within cmake  is not yet in a good state natively.